src/Model/Factory/RencontreDetailsFactory.php

```diff
@@ -190,20 +192,22 @@ private function getParties(array $data): array
     {
         $parties = [];
         foreach ($data as $partieData) {
-            $setDetails = explode(' ', $partieData['detail']);
+            $setsDetails = array_map(function ($setDetail) {
+                return intval($setDetail);
+            }, explode(' ', trim($partieData['detail'])));
```

Comment:
Something like that ?
```suggestion
            $setsDetails = array_map(fn (string $setDetail): int => (int) $setDetail, explode(' ', trim($partieData['detail'])));
```